packages/office-ui-fabric-react/src/components/Callout/Callout.Props.ts

```diff
@@ -181,4 +182,15 @@ export interface ICalloutProps extends React.Props<Callout | CalloutContent> {
    * @deprecated
    */
   targetElement?: HTMLElement;
+
+  /**
+   * One option is fixed below the options in the callout
+   */
+  calloutButton?: ICalloutButton;
```

Comment:
Another approach would be to add an onRender* helper to tweak the container. 

This approach has a lot of problems and we should think of a better way to do what you want to achieve. I'm ok with a custom render override to enable a custom scenario, but making it so specific to a "callout button" seems really specific for a generalized component.